### DISCUSSION TITLE
Translate the tip of the play project button as run project

### DISCUSSION THREAD

**Discussion by CodeLazier** [godotengine/godot-proposals#5223] [Editor]
Run the project in english should be unified, in the project manager is to run, in the project editor is the play project, it is recommended to translate uniformly as run project, because godot also has many application projects and play project are mostly used in the codeless game editor, and obviously godot is a more professional game engine.

by the way: why there are no related functions such as run project in the main menu, this is not in line with habit. especially on Windows.

**Comment by Calinou** [comment]
I opened a pull request for this: https://github.com/godotengine/godot/pull/64756
